lib/ui/window/window.cc

```diff
@@ -332,9 +332,20 @@ void Window::ReportTimings(std::vector<int64_t> timings) {
     return;
   tonic::DartState::Scope scope(dart_state);
 
+  Dart_Handle data_handle = Dart_NewTypedData(Dart_TypedData_kInt64, timings.size());
+
+  Dart_TypedData_Type type;
+  void* data = nullptr;
+  intptr_t num_bytes = 0;
+  FML_CHECK(!Dart_IsError(
+      Dart_TypedDataAcquireData(data_handle, &type, &data, &num_bytes)));
+
+  memcpy(data, timings.data(), sizeof(int64_t) * timings.size());
+  Dart_TypedDataReleaseData(data_handle);
```

Comment:
Consider checking the result of Dart_TypedDataReleaseData call.

Reply:
Done.